app.js

```diff
@@ -0,0 +1,16 @@
+//Importing from html through querySelector
+const form = document.querySelector("form");
```

Comment:
```suggestion
const form = document.querySelector('form');
```
U JS fajlovima koristimo jednostruke navodnike. Treba ispraviti u celom fajlu